WordPress/gutenberg
# Block settings: Update variant of "Apply globally" Button component to secondary

## What?
Fixes: #61834

## Why?
The prominence of "Apply globally" button stands out even more when we have now the option to enable/disable overrides.

## How?
This PR changes the variant of `Apply globally` button to secondary

## Testing Instructions

1. Activate any FSE theme. For example TT4
2. Go to Appearance > Editor
3. Go to Patterns > My Patterns and click on any pattern to edit it.
4. On the right-hand side, go to the Advanced panel and see the `Apply globally` button at the bottom. (Make sure to change any styles so that the button comes in active state)

## Screenshots
<img width="280" alt="image" src="https://github.com/WordPress/gutenberg/assets/77401999/cd42eca8-6f42-49f5-b666-cf9e8cb53f6d">
